src/Appender/AppenderSkeleton.cs

```diff
@@ -321,7 +321,7 @@ public void DoAppend(LoggingEvent loggingEvent)
 				{
 					ErrorHandler.Error("Failed in DoAppend", ex);
 				}
-#if !MONO && !NET_2_0 && !NETSTANDARD1_3
+#if !(MONO || NET_2_0 || NETSTANDARD1_3 || NETSTANDARD2_0)
```

Comment:
I see a naming scheme difference here. According to the conventions so far, a underscore has been there before the version of a target. Thus `NETSTANDARD1_3` should probably read `NETSTANDARD_1_3` and `NETSTANDARD2_0` should become `NETSTANDARD_2_0`. Of course this has slipped through when `netstandard-1.3` support was added.